docs/guide/index.md

```diff
@@ -104,7 +104,10 @@ vitest related /src/index.ts /src/hello-world.js
 | `-w, --watch` | Watch mode |
 | `-o, --open` | Open UI (default: false) |
 | `-t, --testNamePattern <pattern>` | Run tests with names matching the pattern |
-| `--api` | Listen to port and serve API |
+| `--api` | Listen to default port and serve API |
+| `--port <port>` | Listen on custom port: --api required |
```

Comment:
shouldn't we just enable `api`, if some of this is passed?

or is it not intuitive?

Reply:
if we do, than it is quite intuitive :) https://github.com/cacjs/cac#dot-nested-options

Reply:
Oh, that looks great! What do you think @userquin ?

Reply:
> shouldn't we just enable `api`, if some of this is passed?
> 
> or is it not intuitive?

I have this behavior but @patak-dev wants to have `--api`, see https://github.com/vitest-dev/vitest/pull/399#discussion_r777137853

Reply:
I'll change also the `cli` since we can use `--api 3001` as a shotcut for `--api --port 3001`, just add `[port]` option to `--api` entry.